devlib/utils/android.py

```diff
@@ -231,8 +231,10 @@ def methods(self):
             parser = etree.XMLParser(encoding='utf-8', recover=True)
             xml_tree = etree.parse(StringIO(dump), parser)
 
-            package = next((i for i in xml_tree.iter('package')
-                           if i.attrib['name'] == self.package), None)
+            package = None
```

Comment:
Isn't there a better criteria than just taking the first or last item ? It seems pretty arbitrary and not particularly stable, 6 months from now it might get reversed again or even become random.

It would be a better idea to just change the `if` clause's condition to select the item that actually contains the info we need

Reply:
You're right, I will make the changes.